src/languages/en.ts

```diff
@@ -4685,6 +4686,8 @@ const translations = {
                 other: `removed you from ${joinedNames}'s approval workflows and workspace chats. Previously submitted reports will remain available for approval in your Inbox.`,
             };
         },
+        demotedFromWorkspace: ({policyName, oldRole}: DemotedFromWorkspaceParams) =>
+            `updated your role in ${policyName} from ${oldRole} to user. You have been removed from all submitter workspace chats except for you own.`,
```

Comment:
Was this copy reviewed by marketing? (same for the spanish version)

Reply:
For the english bit I followed the copy that is on the opening issue. 
For the spanish version, the checklist mentions I need to post it in #expensify-open-source and it was approved by an internal Expensify engineer, right? What's the procedure to join?

Reply:
Ah nice, so I assume the english copy was already reviewed and you are in luck because I can review the spanish one 😄

Reply:
Nice, thanks!